docs/contributing.md

```diff
@@ -163,26 +163,24 @@ To use a preview build for a package within a project, you need to override the
+   Run `yarn create-release-branch`. This command creates a release branch named `release/<new release version>` and pre-updates the changelogs for all packages with changes. It also generates a file and opens it in your editor, allowing you to specify which packages you want to include in the next release and their versions. Follow the instructions at the top of the file to update it accordingly.
 
-   When you're ready to continue, save and close the file.
+2. **Review and update changelogs for relevant packages.**
 
-2. **Update changelogs for relevant packages.**
+   First, review all pre-updated changelogs to understand the recent changes. Then, for the packages you intend to release, update these sections in each changelog:
 
-   At this point you will be on a new release branch, and a new section will have been added to the changelog of each package you specified in the previous step.
-
-   For each changelog, review the new section and make the appropriate changes:
-
-   - Move each entry into the appropriate category (review the ["Keep a Changelog"](https://keepachangelog.com/en/1.0.0/#types) spec for the full list of categories and the correct ordering of all categories).
-   - Remove any changelog entries that don't affect consumers of the package (e.g. lockfile changes or development environment changes). Exceptions may be made for changes that might be of interest despite not having an effect upon the published package (e.g. major test improvements, security improvements, improved documentation, etc.).
+   - Categorize entries appropriately following the ["Keep a Changelog"](https://keepachangelog.com/en/1.0.0/) guidelines.
+   - Remove changelog entries that don't affect consumers of the package (e.g. lockfile changes or development environment changes). Exceptions may be made for changes that might be of interest despite not having an effect upon the published package (e.g. major test improvements, security improvements, improved documentation, etc.).
    - Reword changelog entries to explain changes in terms that users of the package will understand (e.g., avoid referencing internal variables/concepts).
-   - Consolidate related changes into one change entry if it makes it easier to comprehend.
+   - Consolidate related changes into single entries where appropriate.
 
-   Run `yarn changelog:validate` to check that all changelogs are correctly formatted.
+   Run `yarn changelog:validate` to ensure all changelogs are correctly formatted.
 
-   Commit and push the branch.
+   After finalizing the changelogs and ensuring the release specification is valid, save and close the file. The tool will automatically commit all the changes to the release branch. Note that any changes made to the changelogs of packages not included in the release will be discarded.
+
+   If everything looks in order, push the branch.
```

Comment:
That's a good idea, although I've made some suggestions to reduce the steps here, so that may or may not be necessary, depending on what we end up with. At the very least, it seems like pushing the branch could go in its own step. We could also move it to the next step (because you have to push a branch to create a pull request). What do you think?